src/arith_uint256.cpp

```diff
@@ -10,6 +10,7 @@
 
 #include <stdio.h>
 #include <string.h>
+#include "crypto/common.h"
```

Comment:
I'm sorry to hear that you don't find fun in many of the much more interesting things you've been doing before, but no - we removed this from the policy because the cost is just not worth the benefits. It requires too much time from maintainers and people writing patches for a very tiny (but non-zero) benefit, so we reverted it for that reason.